client/src/Movies/Movie.js

```diff
@@ -30,27 +32,25 @@ const Movie = (props) => {
     return <div>Loading movie information...</div>;
   }
 
-  const { title, director, metascore, stars } = movie;
+  
```

Comment:
I see you deleted the destructuring and the used basic dot notation.  Why the change?